### ISSUE TITLE
Cloudinary Node JS SDK integration error

### ISSUE THREAD

**Issue by ekediala** [cloudinary/cloudinary_npm#286]
Trying to integrate Cloudinary Node SDK for uploading images in a Vue app. However, as soon as I  import cloudinary into any component, it (the component) ceases to work and I get this error in dev tools: 

`TypeError: Cannot read property 'split' of undefined
    at Object.eval (cloudinary.js?9447:1)
    at eval (cloudinary.js:3)
    at Object../node_modules/cloudinary/cloudinary.js (vendors~admin~public.js:540)
    at __webpack_require__ (app.js:767)
    at fn (app.js:130)
    at eval (utilities.js?9bf5:1)
    at Module../src/utilities.js (admin~public.js:963)
    at __webpack_require__ (app.js:767)
    at fn (app.js:130)
    at eval (cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/vuetify-loader/lib/loader.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/blog/BlogPost.vue?vue&type=script&lang=js&:3)`

If I remove the import, the component starts working fine again.

**Comment by ekediala** [comment]
Sorry. Found out it was a problem with Vue.

**Comment by chrisnore94** [comment]
> Sorry. Found out it was a problem with Vue.

Greetings!

Could you tell me, if you remember, how did you get solved this issue? I'm facing the same issue, but trying to integrate "cloudinary" library in an Ionic React Project.

If you can answer me that question, I'd appreciate you!

**Comment by ekediala** [comment]
> > Sorry. Found out it was a problem with Vue.
> 
> Greetings!
> 
> Could you tell me, if you remember, how did you get solved this issue? I'm facing the same issue, but trying to integrate "cloudinary" library in an Ionic React Project.
> 
> If you can answer me that question, I'd appreciate you!

Oh. Sorry. I can't even remember the project this happened on. Haven't written Vue since 2019.

**Comment by chrisnore94** [comment]
Oh! :( Ok! Thanks anyway! Greetings!

**Comment by patrick-tolosa** [comment]
@chrisnore94 if you can open a new issue and describe what you're experiencing we'll be happy to help.

Please include as much information as possible (Your build process, configuration if possible), a reproducible repo is ideal.
Note that this issue is a bit misleading, it's not about using `Node SDK` with `Vue` - Vue is a client side framework while the NodeSDK is server side oriented and the connection here is not clear.

Long story short - open an issue and we'll be happy to assist.
